### ISSUE TITLE
Network.framework usage

### ISSUE THREAD

**Issue by icservis** [HealthTap/swift-nio-mqtt#64]
I see you use Network.framework for supporting TLS, but it limits usage for Foundation systems. I would like to port this library to Rapsbian, but cant compile due to usage of Network.framework. Do you have any plans for replacing this framework with NIOSSL? Thanks in advance

**Comment by bofeizhu** [comment]
Yeah, this is on our roadmap. But it is a lower priority for us. We only aim to support Apple devices for now. We'd be happy to accept PRs if you would like to contribute.

**Comment by icservis** [comment]
Hello,
I spent some time by discovering problem, but I looks like the biggest problem is NIO Transport services, which is used for MQTTClient.
NTS has dependency to Network.framework, this is the part od iOS, MacOS, TVOs.
Right now I don’t see the solution without swift replacement of Network.framework, or, we need remove dependency on NTS

Best regards, Libor
